TriblerGUI/widgets/trustpage/js/help_page.js

```diff
@@ -0,0 +1,329 @@
+function showHelpPage () {
```

Comment:
Furthermore, make a method .initialize() in which you draw the entire help page (but don't display it). This way it won't be redrawn every time you click 'Help'